Artemis/Artemis/Models/Profiles/ProfileModel.cs

```diff
@@ -84,7 +84,10 @@ public void FixOrder()
                 c.Pop();
             }
 
-            return ImageUtilities.DrawinVisualToBitmap(visual, keyboardRect);
+            //TODO DarthAffe 13.06.2016: This is just to make it work
+            //It would be better to somehow draw directly but I don't really understand dat visual stuff right now :p

```

Comment:
DrawingVisual is basically WPF's version of Graphics. Since WPF stuff is all vector based it gets translated to a (pixel based) bitmap here so it can be drawn to the keyboards.